deploy/olm-catalog/pulp-operator/manifests/pulp-operator.clusterserviceversion.yaml

```diff
@@ -346,6 +346,11 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: DB fields encryption
+        path: db_fields_encryption_secret
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:hidden
```

Comment:
This will make this only accessible using YAML, no selection from the form based UI. Is that what you want?

Otherwise you could keep it in advanced but add:
```
- urn:alm:descriptor:io.kubernetes:Secret
```
instead of hidden.

Reply:
I want it to be accessible on the UI, I'll change it, thank you!